setup.py

```diff
@@ -154,7 +155,9 @@
           'scipy >= 1.3.0',
           'h5py >= 2.9.0',
           'hdmedians >= 0.14.1',
-          'scikit-learn >= 0.19.1'
+          'scikit-learn >= 0.19.1',
+          'cython',
```

Comment:
It looks like we're still in a mixed C/Cython situation. The extensions will still compile without Cython and it appears you've updated all the C code.

Should Cython be skipped as a dependency here? It won't be used unless `USE_CYTHON` is true during installation.

Reply:
Removed, thanks!